tests/selectors_test.py

```diff
@@ -96,7 +88,7 @@ def test_set_ops(
     expected: list[str],
     request: pytest.FixtureRequest,
 ) -> None:
-    if ("pyspark" in str(constructor)) or "duckdb" in str(constructor):
+    if "duckdb" in str(constructor) and not expected:
```

Comment:
To figure this out due to:

> # TODO(marco): return empty relation with 0 columns?
>            return self._from_native_frame(self._native_frame.limit(0))

took me some sanity 😂

I tried in different ways to create an empty table but duckdb doesn't really like that. Maybe we might just raise an error?